src/app/view/components/_common/dot-action-button/dot-action-button.component.ts

```diff
@@ -8,15 +9,46 @@ import { MenuItem } from 'primeng/primeng';
  * @class ActionButtonComponent
  */
 @Component({
-    encapsulation: ViewEncapsulation.None,
     selector: 'dot-action-button',
     styleUrls: ['./dot-action-button.component.scss'],
     templateUrl: 'dot-action-button.component.html'
 })
 
-export class DotActionButtonComponent {
-    @Input() model?: MenuItem[];
-    @Input() command?: ($event) => void;
+export class DotActionButtonComponent implements OnInit {
+    @ViewChild('menu') menu: Menu;
+
+    @Input() disabled: boolean;
+    @Input() icon: string;
     @Input() label: string;
-    @Input() flat: boolean;
```

Comment:
Because the nature of this component if we pass the label we need to make it flat.